orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/sigs/SignatureAllocation.java

```diff
@@ -16,4 +16,7 @@ public interface SignatureAllocation
     double allocation();
 
     double percent();
+
+    @Nullable
```

Comment:
Why would this ever be null? 

I can imagine there is no etiology for `MISALLOC` but maybe you can assign "-" in the datamodel in that case? Since etiology is not for display purpose anyways I think I would avoid null

Reply:
Indeed my reasoning behind making it nullable was because of `MISALLOC`